platform/vs/docker-sonic-vs/chassis_db.py

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env python
```

Comment:
I'm not totally sure I understand this script. Is this supposed to mock the presence of the database_chassis container?

Reply:
the script populates system port configuration into chassis_db. The chassis_db.json file is considered as part of next virtual-chassis PR and not in this PR, here is a part of the file:
cat device/virtual/x86_64-kvm_x86_64-r0/virtual_chassis/0/chassis_db.json          
{                                                                                                                                                             
    "SYSTEM_PORT|Linecard1|Ethernet0": {                                                                                                                  
        "speed": "40000",                                                                                                                          
        "system_port_id": "1",                                                                                                                                                                    
        "switch_id": "0",                                                                                                                    
        "core_id": "0",                                                                                                                                                                           
        "core_port_id": "1"                                                                                                                      
    },                                                                                                                                               
    "SYSTEM_PORT|Linecard1|Ethernet4": {                                                                                                                                
        "speed": "40000",                                                                                                                                       
        "system_port_id": "2",                                                                                                                             
        "switch_id": "0",                                                                                                                             
        "core_id": "0",                                                                                                                               
        "core_port_id": "2"                                                                                                                                                                       
    },                                                             
.....

Reply:
Cool, thanks!